app/temp_users/routes.py

```diff
@@ -56,15 +58,45 @@ def validate(token, id_application):
     Call the API to validate a temp user
     """
     data_to_post = {"token": token, "id_application": id_application}
+
+    # Get temp user infos
+    temp_user = (
+        db.session
+            .query(TempUser.confirmation_url)
+            .filter(token == TempUser.token_role)
+            .first()
+    )
+    if not temp_user:
+        return {"msg": "Aucun utilisateur trouvé avec le token user demandé"}, 42
+    url_after_validation = temp_user[0]
+
+    # Call temp user validation URL
     url_validate = (
         current_app.config["URL_APPLICATION"] + "/api_register/valid_temp_user"
     )
     r = req_lib.post(url=url_validate, json=data_to_post, cookies=request.cookies)
+    if r.status_code != 200:
+        flash("Erreur durant la validation de l'utilisateur temporaire", "error")
+        return redirect(url_for("temp_users.temp_users_list"))
+    elif not url_after_validation:
+        flash("L'utilisateur a bien été validé")
+        return redirect(url_for("temp_users.temp_users_list"))
+    
+    user_data = r.json()
+
+    # Call post UsersHub actions URL
+    if url_after_validation:
+        r = req_lib.post(
+            url=url_after_validation, 
+            json=user_data, 
+            cookies=request.cookies
+        )
     if r.status_code == 200:
         flash("L'utilisateur a bien été validé")
         return redirect(url_for("temp_users.temp_users_list"))
     else:
-        flash("Une erreur s'est produite", "error")
+        flash("Erreur durant l'appel des actions de l'application", "error")
+        log.error(f"Error HTTP {r.status_code} for {url_after_validation}")
```

Comment:
Les fstrings necessite python 3.6, on va le passer sur GeoNature, mais c'est pas encore fait sur usershub. En attendant est-ce qu'on peut privilégier le formatage des chaines avec `.format() `?

Reply:
Noté. C'est corrigé dans le dernier commit.